app/audit/auditLog.ts

```diff
@@ -70,18 +71,15 @@ export type EventData =
   | BundesIdentIdentifiedData;
 
 export const saveAuditLog = async (data: AuditLogData) => {
-  await db.auditLog.create({
+  await db.auditLogV2.create({
     data: {
+      user: hash(data.username),
```

Comment:
Why did you decide for Sha-256 instead of bcrypt for hashing?

Reply:
I went for sha256 because

1. it's faster. bcrypt is slower by design and I'm thinking about the implications of this during migration;
2. it's more readily available (e.g. via command line tools) which might be useful since you could easily generate the identifier for the data you're looking for outside of the app logic (i.e. you wouldn't need to fire up the application or right a script just to generate the hash for a given email); and
3. it serves our purpose and fulfills our security requirements. We're not protecting passwords against rainbow tables, we're just making sure that the stored emails are not immediately decipherable.

What would be the justification for using bcrypt instead? bcrypt with random salt is out of the question since it would completely defeat the purpose of the new column (random access with a known email goes out the window). With a fixed salt we'd have some degree of protection against rainbow tables at the cost of speed, convenience, complexity and maintenance.

Reply:
We were using SHA-256 in in couple of places already (e.g. cookie suffixes) and even SHA-1 in others.